### DISCUSSION TITLE
proxmox LXC install can I add dedicated drive for storage?

### DISCUSSION THREAD

**Discussion by danuals** [blakeblackshear/frigate#2075] [Ask A Question]
Is it possible to addd a dedicated drive to a proxmox lxc install? (Portainer) for recording/clips etc, want to use it as 24/7  nvr and my proxmox drive is too small to use. 

any nudge in the right direction would be great 

thanks

**Comment by danuals** [comment]
so I mounted a usb hdd to proxmox. and mapped volume paths for frigate folders to it in the lxc container. seems to be saving all clips now to the dedicated hdd

**Reply by mkono87** [reply]
Assuming you use used mountpoint eg mp0   this is the way :)
